CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 * Removed the validation of a subtype change in integrations and scripts from **validate**.
+* Added a validation to the **validate** command to make sure we are using the https protocol in the defaultvalue of the url parameter.
```

Comment:
```suggestion
* Added a **validate** check, making sure `https`-prefix values are used when setting defaultvalue, rather than the discuraged `http`.
```